Formula/wasm-micro-runtime.rb

```diff
@@ -0,0 +1,42 @@
+class WasmMicroRuntime < Formula
+  desc "WebAssembly Micro Runtime (WAMR)"
+  homepage "https://github.com/bytecodealliance/wasm-micro-runtime"
+  url "https://github.com/bytecodealliance/wasm-micro-runtime/archive/refs/tags/WAMR-05-18-2022.tar.gz"
+  version "2022.05.18"
+  sha256 "c71f571f99c7028fd7595126ec25d6df4bc9ba62702bb88cbd5ea8a29e477f12"
+  license "Apache-2.0"
+  head "https://github.com/bytecodealliance/wasm-micro-runtime.git", branch: "main"
+
+  depends_on "cmake" => :build
+
+  def install
+    cmake_args = std_cmake_args + %w[
+      -DWAMR_BUILD_MULTI_MODULE=1
+      -DWAMR_BUILD_DUMP_CALL_STACK=1
+      -DWAMR_BUILD_JIT=0
+      -DWAMR_BUILD_LIBC_UVWASI=0
+    ]
+
+    cmake_source = "product-mini/platforms/#{OS.mac? ? "darwin" : "linux"}"
+
+    system "cmake", "-S", cmake_source, "-B", "build", *cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+
+    rm_r "build/"
+
+    system "cmake", "-S", ".", "-B", "build", *cmake_args
```

Comment:
Given the suggestion above:
```suggestion

    system "cmake", "-S", ".", "-B", "build", *cmake_args, *std_cmake_args
```